conf.d/main

```diff
@@ -28,3 +28,9 @@ rm -rf /var/www/{js,css,images}
 
 # fix pkitool bug
 sed -i 's|\(KEY_ALTNAMES=\)"$KEY_CN"|\1"DNS:$KEY_CN"|' /etc/openvpn/easy-rsa/pkitool
+
+# temporary fix to https://github.com/turnkeylinux/tracker/issues/1200
+ln -s /etc/openvpn/easy-rsa/openssl-1.0.0.cnf /etc/openvpn/easy-rsa/openssl.cnf
```

Comment:
@qrntz - do you reckon your could have a quick look at this one. IMO it's probably fine, although would appreciate a second opinion. Cheers.

Reply:
According to my understanding of https://github.com/OpenVPN/easy-rsa/commit/e8cd6c980742bb86c34286e655346778275cc36d and the buster `easy-rsa` package this is perfectly fine. The [diff](https://gist.github.com/qrntz/f93cfc12c2aa4a27f8a5bd29e23abec0) between `stretch` `openssl-1.0.0.cnf` and upstream/`buster` `openssl-easyrsa.cnf` includes no changes that are OpenSSL settings-specific rather than easy-rsa behavior-specific.